lms/djangoapps/certificates/tests/test_webview_views.py

```diff
@@ -222,6 +222,103 @@ def test_rendering_course_organization_data(self):
         self.assertIn('logo_test1.png', response.content)
 
     @override_settings(FEATURES=FEATURES_WITH_CERTS_ENABLED)
+    @patch.dict("django.conf.settings.SOCIAL_SHARING_SETTINGS", {
+        "CERTIFICATE_TWITTER": True,
+        "CERTIFICATE_FACEBOOK": True,
+    })
```

Comment:
There are existing test for testing LinkedIn share feature. This particular test has microsite enabled and in that case LinkedIn share is not supported.